src/error/specification

```diff
@@ -874,3 +874,15 @@ severity:warning
 ingroup:plugin
 module:resolver
 macro:NOCWD
+
+number:147
+description:issue while interfacing with git
+severity:error
+ingroup:plugin
+module:gitresolver
```

Comment:
its better if you specify `macro:` here and use the macro in the code. Otherwise you likely get difficult to fix merge conflicts when someone else adds errors.